plugins/kn-image/README.md

```diff
@@ -0,0 +1,220 @@
+# Kn service build and deploy plugin
+
+## Prequirement
+Before you begin this task, you should prepare a kubernetes cluster which has been installed tekton-pipeline and access it.
+### Confirm tekton-pipeline has been installed
```

Comment:
Add new line. Be consistent in formatting

Reply:
Change the format to add newline before source-code